graphix/sim/statevec.py

```diff
@@ -1,8 +1,9 @@
+from copy import deepcopy
+
```

Comment:
please remove extra line here.

Reply:
I followed `isort` package here. Why don't we obey PEP8?

Reply:
sure if there's good reason!